### ISSUE TITLE
Hush logo on startup

### ISSUE THREAD

**Issue by leto** [MyHush/SilentDragon#2]
I've tried various things, a GIF seems to be required and has been set, but a blank/white area is currently rendered. Potentially because the height/width of image are not set correctly in QT

**Comment by leto** [comment]
Current plan is to exactly match the height/width of existing logo, so no code has to change, except the file itself, but not the filename.

**Comment by leto** [comment]
DONEZO, waiting on Logo Contest for official logo, which will be another issue
